app/models/case_closure/offender_complaint_appeal_outcome.rb

```diff
@@ -0,0 +1,45 @@
+# == Schema Information
+#
+# Table name: case_closure_metadata
+#
+#  id                      :integer          not null, primary key
+#  type                    :string
+#  subtype                 :string
+#  name                    :string
+#  abbreviation            :string
+#  sequence_id             :integer
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
+#  requires_refusal_reason :boolean          default(FALSE)
+#  requires_exemption      :boolean          default(FALSE)
+#  active                  :boolean          default(TRUE)
+#  omit_for_part_refused   :boolean          default(FALSE)
+#
+
+module CaseClosure
+  class OffenderComplaintAppealOutcome < Metadatum
+    def self.upheld
+      where(abbreviation: 'upheld').singular
+    end
+
+    def self.not_upheld
+      where(abbreviation: 'not_upheld').singular
+    end
+
+    def self.not_response_received
+      where(abbreviation: 'not_response_received').singular
+    end
+
+    def upheld?
+      abbreviation == 'upheld'
+    end
+
+    def not_upheld?
+      abbreviation == 'not_upheld'
+    end
+
+    def not_response_received?
+      abbreviation == 'not_response_received'
```

Comment:
It's probably a "me problem" rather than a problem with the implementation, but just fwiw I find it surprising that "abbreviation" is storing these values. Is that how it's meant to be used?

Reply:
I agreed, I found this a while ago which is puzzled me as well,  but don't want to change the way right now

Reply:
it may have other thought around this field initially e.g. the possibility of sub-group (2 level structure)? but somehow end up with majority of metadata classes  is one level there, and also it explain why it use `.singular`

Reply:
Cool, thanks - sounds sensible! 😁